main.go

```diff
@@ -293,7 +293,7 @@ func doStdInAndParseForIQ(config configuration.IqConfiguration) (err error) {
 		"projectList": mod.ProjectList,
 	}).Debug("Obtained project list")
 
-	var purls = mod.ExtractPurlsFromManifestForIQ()
+	var purls = mod.ExtractPurlsFromManifest()
```

Comment:
Similar thought about separating concerns of IQ vs. OSSSI processing out the `main.go` code. If only that guy working on the Cobra command stuff would hurry up. ;)